src/components/IconState.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <span class="icon-state">
+    <v-icon color="primary" dark @click.stop="dialog = true">{{ localIcon }}</v-icon>
+    <v-dialog v-model="dialog" max-width="400">
+      <v-card>
+        <v-card-title class="headline">Please choose your Icon?</v-card-title>
+        <v-card-text>
+          <v-row>
+            Please enter the name of icon you wish to use for this state. You can get the icon names from
+            https://material.io/resources/icons/ or https://cdn.materialdesignicons.com/5.3.45/.
```

Comment:
these should be clickable links. You may want to see how chips look here. https://vuetifyjs.com/en/components/chips/